src/components/utils/Markdown.tsx

```diff
@@ -12,19 +13,31 @@ export default class Markdown extends React.Component<MarkdownProps> {
 
   constructor(props) {
     super(props);
-    this.renderer = new HtmlRenderer({ safe: true });
```

Comment:
Why you remove `safe`?

Reply:
I've removed it because I use `<mark>` HTML tag for highlighting the search term inside Markdown code. This is allowed by Markdown spec. I've tried to wrap the search term in the final HTML string, but this is not a trivial task. E.g. if the search term is `p`, it breaks the HTML code: `<<mark>p</mark>>Some descri<mark>p</mark>tion text<<mark>p</mark>>`.